aspnetcore/fundamentals/minimal-apis/security.md

```diff
@@ -0,0 +1,197 @@
+
+There are two strategies for determining user access to resources in the authorization layer:
+
+* Role-based strategies determine a user's access based on the role they are assigned, such as `Administrator` or `User`. For more information on role-based authorization, see [role-based authorization documentation](/aspnet/core/security/authorization/roles).
+* Claim-based strategies determine a user's access based on claims that are issued by a central authority. For more information on claim-based authorization, see [claim-based authorization documentation](/aspnet/core/security/authorization/claims).
+
+In ASP.NET, both strategies are captured into an authorization requirement. The authorization service leverages authorization handlers to determine whether or not a particular user meets the authorization requirements applied onto a resource. 
+
+## Enabling authentication in minimal applications
+
+To enable authentication in an application, invoke the [`AddAuthentication`](/dotnet/api/microsoft.extensions.dependencyinjection.authenticationservicecollectionextensions.addauthentication) method to register the required authentication services on the app's service provider.
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthentication();
+
+var app = builder.Build();
+
+app.Run();
+```
+
+Typically, a specific authentication strategy is used. In the following sample, the app is configured with support for JWT bearer-based authentication.
+
+```csharp
+using Microsoft.AspNetCore.Authentication.Cookie;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthentication().AddJwtBearer();
+
+var app = builder.Build();
+
+app.Run();
+```
+
+By default, the  [`WebApplication`](dotnet/api/microsoft.aspnetcore.builder.webapplication) automatically registers the authentication and authorization middlewares if certain authentication and authorization services are enabled. In the following code sample below, it's not necessary to invoke [`UseAuthentication`](dotnet/api/microsoft.aspnetcore.builder.authappbuilderextensions.useauthentication) or [`UseAuthorization`](/dotnet/api/microsoft.aspnetcore.builder.authorizationappbuilderextensions.useauthorization) to register the middlewares since [`WebApplication`](dotnet/api/microsoft.aspnetcore.builder.webapplication) does this automatically after services are added.
```

Comment:
This will fix the build error
```suggestion
By default, the [`WebApplication`](/dotnet/api/microsoft.aspnetcore.builder.webapplication) automatically registers the authentication and authorization middlewares if certain authentication and authorization services are enabled. In the following code sample below, it's not necessary to invoke [`UseAuthentication`](dotnet/api/microsoft.aspnetcore.builder.authappbuilderextensions.useauthentication) or [`UseAuthorization`](/dotnet/api/microsoft.aspnetcore.builder.authorizationappbuilderextensions.useauthorization) to register the middlewares since [`WebApplication`](dotnet/api/microsoft.aspnetcore.builder.webapplication) does this automatically after services are added.
```